dotnet/samples/KernelSyntaxExamples/Plugins/MenuPlugin.cs

```diff
@@ -20,9 +20,19 @@ public string GetSpecials()
 
     [KernelFunction, Description("Provides the price of the requested menu item.")]
     public string GetItemPrice(
-        [Description("The name of the menu item.")]
+            [Description("The name of the menu item.")]
             string menuItem)
     {
         return "$9.99";
     }
+
+    [KernelFunction, Description("Returns true if the kitchen has ran out of the item.")]
+    public bool IsItem86d(
```

Comment:
What is the meaning of `IsItem86d` function name?

Reply:
86'd is restaurant lingo for "out of the item"

Reply:
I'll add remark since I'm doing another comment also